gym_snake/envs/snake_env.py

```diff
@@ -5,7 +5,7 @@
 from gym import spaces
 from numpy import ndarray
 from gym_snake.envs.snakeGameGym import *
-from gym_snake.envs.snakeRewardFuncs import basic_reward_func
+from gym_snake.envs.snakeRewardFuncs import basic_reward_func, reward_closer_to_fruit
```

Comment:
As we build more of these, we may not want to call them out individually in the import statement. Maybe we should `import *`?